app/src/main/java/io/github/cmput301f19t19/legendary_fiesta/ui/SpinnerArrayAdapter.java

```diff
@@ -66,22 +71,35 @@ public View getSpinnerView_DropDown(int position, View convertView, ViewGroup pa
         /*
         if None is the object, we can't use Mood to get the information because Mood doesn't have this moodType
          */
-        if(objects[position].equals("None")){
-            dropDownText.setText("None");
+        if(objects[position].equals( context.getResources().getString(R.string.filter_empty) )){
+            dropDownText.setText(R.string.filter_empty);
         }else{
 
             /*
-             * Get the Mood
+             * Get the Mood by getting the value of the input filter. For example, Angry will give you 3.
              */
             String moodName = objects[position].toUpperCase();
             Integer moodType = Mood.MoodTypes.get(moodName);
-            Mood mood = new Mood(moodType);
+            Mood mood = null;
+
+            try {
+                 mood = new Mood(moodType);
+            }catch (NullPointerException e){
+                Log.d("Error","Nullpointer exception for Filter " + e);
```

Comment:
Use a more severe log level here:

```suggestion
                Log.e("FeelsLog", "Exception in Filter ", e);
```